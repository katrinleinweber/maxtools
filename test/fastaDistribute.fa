>testseq1|file1|bla
blababab
balblab
balbal

>testseq2|file2|blabla
vbalbalbjk
>testseq3|file1|blab
balbal
balba

