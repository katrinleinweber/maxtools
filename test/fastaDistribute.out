
94 file1.fa
49 file2.fa
